dev/com.ibm.ws.microprofile.config/src/com/ibm/ws/microprofile/config/archaius/composite/CompositeConfig.java

```diff
@@ -85,9 +86,9 @@ public void onConfigUpdated() {
      * @return
      */
     private PollingDynamicConfig addConfig(ConfigSource source, ScheduledExecutorService executor, long refreshInterval) {
-        //if it is an internal config source then it should not refresh
+        //if it is an internal static config source then it should not refresh
         //this is a hack for now ... dynamic config sources will be fixed properly in the next version
```

Comment:
> this is a hack for now

Is this still a hack?

Reply:
yes ... i should not have had to invent my own internal marker interfaces, there should be something on the spi that tells me